src/main/java/eu/darkbot/api/objects/Location.java

```diff
@@ -4,42 +4,49 @@
     /**
      * Returns distance to location ox & oy
      */
-    default double distanceTo(double ox, double oy) {
-        ox -= getX();
-        oy -= getY();
-        return Math.sqrt(ox * ox + oy * oy);
-    }
+    double distanceTo(double ox, double oy);
```

Comment:
i think this one can be defaulted, the distance between one location and another is a pretty universal concept as long as you provide x and y for both